### PR TITLE
feat: Add Arseeding support

### PR DESCRIPTION
**Pull request by pawanpaudel93**
## Summary

This PR adds ArSeeding support to sync repositories to Protocol Land.

## How to test
1) Clone repo and checkout `feat/arseeding-support` branch 
2) Install and build: `pnpm install && pnpm build`
3) Install the package globally: `npm install -g .`
4) Deposit some amount of supported Arweave tokens to [EverPay](https://everpay.io/). Supported tokens are: `['XYZ', 'ARDRIVE', 'PIA', 'VRT', 'U', 'STAMP', 'AR']`
5) Add the following environment variables to GH action or .env:

```dosini
WALLET=''
REPO_TITLE=''
REPO_DESCRIPTION=''
STRATEGY='ARSEEDING'
ARSEEDING_TOKEN_SYMBOL='AR'
```
6) Follow the steps in [README.md](https://github.com/labscommunity/protocol-land-sync/tree/feat/arseeding-support?tab=readme-ov-file#locally-without-github-action)

### CODE REVIEW

File: src/lib/arseeding.ts
Comment (7i7o) [inline src/lib/arseeding.ts:1]: Is there an option for arseeding-js to omit logging the key to console? That will be saved in the gh action logs, exposing it for anyone with access to it

**Review by 7i7o — CHANGES_REQUESTED** [review]
tACK
It works. Arseeding is a little slower than turbo or irys, so the repo can take a minute or two to show correctly in PL.
Left a comment on the `arseeding.ts` file for important changes

File: src/lib/arseeding.ts
Comment (pawanpaudel93) [reply src/lib/arseeding.ts:1]: So i figured out where the logging is occuring. It's occuring in `everpay-js` sdk. I have now silenced the logging from `arseeding.ts`.

**Review by kranthicodes — DISMISSED** [review]
tACK

**Review by 7i7o — DISMISSED** [review]
tACK
LGTM! 🚀

**Review by 7i7o — APPROVED** [review]
LGTM! 🚀

**Pull request merged**